docs/source/CustomizingTheWorkflow/DASystem.rst

```diff
@@ -151,7 +155,7 @@ Users may find the following example ``yaml`` configuration file to be a helpful
 
 .. note::
 
-   Any default values indicated in the sections below are the defaults set in ``letkfoi_snow.yaml`` or ``GHCN.yaml`` (found within the ``land-offline_workflow/DA_update/jedi/fv3-jedi/yaml_files/release-v1.0/`` directory).
+   Any default values indicated in the sections below are the defaults set in ``letkfoi_snow.yaml`` or ``GHCN.yaml`` (found within the ``land-DA_workflow/DA_update/jedi/fv3-jedi/yaml_files/20220921`` directory).
```

Comment:
@gspetro-NOAA this should be changed to 'DA_update/jedi/fv3-jedi/yaml_files/psl_develop'